apps/elewa-group-website/src/app/app.module.ts

```diff
@@ -5,14 +5,11 @@ import { ScullyLibModule } from '@scullyio/ng-lib';
 
 import { AppComponent } from './app.component';
 
-import { HomePageModule } from '@elewa-group/pages/elewa/home'
+import { HomePageModule } from '@elewa-group/pages/elewa/home';
```

Comment:
No need to change this file formatting.

The formatting of the file was correct en is no lomger correct